hc/api/tests/test_send_reports.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
```

Comment:
@jimmykimani All the changes you have made so far are related to the `hc/accounts` folder. Any reason why the tests located in `hc/api`?